### ISSUE TITLE
The relevance of mAP and error during training.

### ISSUE THREAD

**Issue by Jin-S-Kim** [AlexeyAB/darknet#3688]
Hello AlexeyAB. I'm training YOLO by using this command in terminal.
./darknet detector train data/obj.data yolo-obj.cfg darknet53.conv74
![image](https://user-images.githubusercontent.com/31817047/62185700-6a3b8100-b39e-11e9-94dc-512d70aa3af7.png)

I thought the mAP would be the lowest when the error was the lowest, but the 13500th epoch had a lower error than the 9000th epoch, but the mAP was 91%.(On the other hand, 9000th eopch's mAP was 92%).
Is training going well??

And your repository is very convenient and it stores the best weights.
![image](https://user-images.githubusercontent.com/31817047/62185822-dae29d80-b39e-11e9-9b25-d6ca2bfaf2fa.png)

Do you know if this is the weight of the epoch?

**Comment by LukeAI** [comment]
you should stop training when AP stops going up. The loss will keep going down but if AP is not still going up then it is overfitting. Read more here: https://towardsdatascience.com/preventing-deep-neural-network-from-overfitting-953458db800a
